jenkins-webhook/1.0.0/README.md

```diff
@@ -0,0 +1,12 @@
+# Jenkins Webhook Integration
+
+- Using Keptn version 0.10.* to 0.12.x? Use the [Keptn Jenkins Shared Library](https://artifacthub.io/packages/keptn/keptn-integrations/jenkins-library).
+- Using a Keptn version above 0.12.x? Use this integration method.
+
+Keptn versions over 0.12.x include the webhook service as standard. Use this to trigger Jenkins either asynchronously with no changes required to your Jenkins job. 
+
+Alternatively trigger Jenkins synchronously with one change to the job.
+
+This video explains both asynchronous and synchronous methods:
+
+[![Keptn Jenkins Webhook Integration](https://img.youtube.com/vi/ehI23d7s-dY/0.jpg)](https://www.youtube.com/watch?v=ehI23d7s-dY&t=50s)
```

Comment:
I highly appreciate making a YouTube video for this! :+1:

Though I would also appreciate if you could add some basic instructions in here, similar to what we have with [Github actions](https://github.com/keptn-contrib/artifacthub/tree/main/githubaction/1.0.0)